google/resource_compute_ssl_policy_test.go

```diff
@@ -0,0 +1,308 @@
+				Config: testAccComputeSslPolicyTlsVersion(sslPolicyName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeSslPolicyExists(
+						"google_compute_ssl_policy.tlsversion", &sslPolicy),
+					resource.TestCheckResourceAttr(
+						"google_compute_ssl_policy.tlsversion", "min_tls_version", "TLS_1_2"),
+				),
+			},
+			resource.TestStep{
+				ResourceName:      "google_compute_ssl_policy.tlsversion",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccComputeSslPolicy_custom(t *testing.T) {
+	t.Parallel()
+
+	var sslPolicy computeBeta.SslPolicy
+	sslPolicyName := fmt.Sprintf("test-ssl-policy-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeSslPolicyDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccComputeSslPolicyCustom(sslPolicyName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeSslPolicyExists(
+						"google_compute_ssl_policy.custom", &sslPolicy),
+					resource.TestCheckResourceAttr(
+						"google_compute_ssl_policy.custom", "min_tls_version", "TLS_1_2"),
+					resource.TestCheckResourceAttr(
+						"google_compute_ssl_policy.custom", "profile", "CUSTOM"),
+					testAccCheckComputeSslPolicyCustomFeatures(&sslPolicy, "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384"),
+				),
+			},
+			resource.TestStep{
+				ResourceName:      "google_compute_ssl_policy.custom",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccCheckComputeSslPolicyExists(n string, sslPolicy *computeBeta.SslPolicy) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("No ID is set")
+		}
+
+		config := testAccProvider.Meta().(*Config)
+
+		project, err := getTestProject(rs.Primary, config)
+		if err != nil {
+			return err
+		}
+
+		name := rs.Primary.Attributes["name"]
+
+		found, err := config.clientComputeBeta.SslPolicies.Get(
+			project, name).Do()
+		if err != nil {
+			return fmt.Errorf("Error Reading SSL Policy %s: %s", name, err)
+		}
+
+		if found.Name != rs.Primary.ID {
+			return fmt.Errorf("SSL Policy not found")
+		}
+
+		*sslPolicy = *found
+
+		return nil
+	}
+}
+
+func testAccCheckComputeSslPolicyDestroy(s *terraform.State) error {
+	config := testAccProvider.Meta().(*Config)
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "google_compute_ssl_policy" {
+			continue
+		}
+
+		_, err := config.clientComputeBeta.SslPolicies.Get(
+			config.Project, rs.Primary.ID).Do()
+		if err == nil {
+			return fmt.Errorf("SSL Policy still exists")
+		}
+	}
+
+	return nil
+}
+
+func testAccCheckComputeSslPolicyCustomFeatures(sslPolicy *computeBeta.SslPolicy, n string) resource.TestCheckFunc {
```

Comment:
You can actually get rid of this test function- the import check will verify this on its own (it'll call read and then try to populate state from it and compare it to the config, so if custom_features wasn't set upstream then the import would fail)